src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -1564,9 +1565,12 @@ private static bool NormalizeTaskTypesInPointer(CSharpCompilation compilation, r
             }
             if (type.ContainsNullableReferenceTypes())
             {
-                // PROTOTYPE(NullableReferenceTypes): Create attribute
-                // with PEModuleBuilder.SynthesizeNullableAttribute().
-                // See AttributeTests_Nullable.EmitAttribute_Interface.
+                // PROTOTYPE(NullableReferenceTypes): Not including top-level nullability.
+                var attr = moduleBuilder.SynthesizeNullableAttribute(type, TypeSymbolWithAnnotations.Create(type, isNullableIfReferenceType: null));
```

Comment:
>var [](start = 16, length = 3)

Please spell out the type.

Reply:
Matches the code above.

---
In reply to: [194565869](https://github.com/dotnet/roslyn/pull/27589#discussion_r194565869) [](ancestors = 194565869)

Reply:
> Matches the code above.

Sorry, this doesn't make it easier to understand what the code is doing. this is a new code and it doesn't follow our team's coding style. I think the readers would appreciate if you change the code above too.

---
In reply to: [194615912](https://github.com/dotnet/roslyn/pull/27589#discussion_r194615912) [](ancestors = 194615912,194565869)